app/bundler/strings.ts

```
@@ -0,0 +1,13 @@
+// matches ANSI escape sequences (colors, cursor movement, etc.)
+// oxlint-disable-next-line no-control-regex
+const ANSI_REGEX = /\x1b\[[0-9;]*[a-z]/gi
```

Biome (2.3.13)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)